### ISSUE TITLE
Ubuntu-Build does not finish in packer v1.6.4/1.6.5

### ISSUE THREAD

**Issue by Runkar** [hashicorp/packer#10236]
Hello,

my Ubuntu 20.04 packer build seems to encounter an error implemented after packer v1.6.0:
While "Waiting for IP" packer suddenly stops the build process at about 1m 30s without giving me some error hints...

This error only occurs with packer v1.6.4+, with packer v1.6.0 the build is flawless.

Full debug log:

````
2020/11/10 08:26:35 [INFO] Packer version: 1.6.5 [go1.15.3 linux amd64]
2020/11/10 08:26:35 [DEBUG] Discovered plugin: windows-update = /opt/packer_v1.6.5/packer-provisioner-windows-update
2020/11/10 08:26:35 using external provisioners [windows-update]
2020/11/10 08:26:35 Checking 'PACKER_CONFIG' for a config file path
2020/11/10 08:26:35 'PACKER_CONFIG' not set; checking the default config file path
2020/11/10 08:26:35 Attempting to open config file: /root/.packerconfig
2020/11/10 08:26:35 [WARN] Config file doesn't exist: /root/.packerconfig
2020/11/10 08:26:35 Setting cache directory: /opt/git/packer-config/ubuntu/packer_cache
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value: /opt/git/packer-config/ubuntu/bash_scripts/apt_update.sh
2020/11/10 08:26:35 Deleting kv.Value: 3s
2020/11/10 08:26:35 Deleting kv.Value: clashare1003
2020/11/10 08:26:35 Deleting kv.Value: Autor: Jens Schütz, erstellt am: 04.05.2020, zuletzte geändert am: 04.05.2020
2020/11/10 08:26:35 Deleting kv.Value: 1
2020/11/10 08:26:35 Deleting kv.Value: true
2020/11/10 08:26:35 Deleting kv.Value: sos_fas1_01_nfs_vol1_encrypted
2020/11/10 08:26:35 Deleting kv.Value: Variablen für Packer-Datei zur automatisierten Erstellung des Ubuntu 20.04 Templates
2020/11/10 08:26:35 Deleting kv.Value: pvscsi
2020/11/10 08:26:35 Deleting kv.Value: true
2020/11/10 08:26:35 Deleting kv.Value: 512
2020/11/10 08:26:35 Deleting kv.Value: 10240
2020/11/10 08:26:35 Deleting kv.Value: bios
2020/11/10 08:26:35 Deleting kv.Value: test
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value: /opt/git/packer-config/ubuntu/ubuntu_20.04/http
2020/11/10 08:26:35 Deleting kv.Value: true
2020/11/10 08:26:35 Deleting kv.Value: 10m
2020/11/10 08:26:35 Deleting kv.Value: DEV99|PACKER|PACKER_EPG
2020/11/10 08:26:35 Deleting kv.Value: vmxnet3
2020/11/10 08:26:35 Deleting kv.Value: Ubuntu Linux 20.04

- erstellt: {{isotime "02.01.2006"}}
- 512MB / 10GB HDD
- SSH aktiv
- open-vm-tools, jq, chrony
2020/11/10 08:26:35 Deleting kv.Value: [sos_fas1_01_nfs_vol1_encrypted] ISO/ubuntu-20.04-live-server-amd64.iso
2020/11/10 08:26:35 Deleting kv.Value: ubuntu64Guest
2020/11/10 08:26:35 Deleting kv.Value: /opt/git/packer-config/ubuntu/bash_scripts/post_install_cleanup.sh
2020/11/10 08:26:35 Deleting kv.Value: /opt/git/packer-config/ubuntu/bash_scripts/post_install_config.sh
2020/11/10 08:26:35 Deleting kv.Value: true
2020/11/10 08:26:35 Deleting kv.Value: 2048
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value: 15m
2020/11/10 08:26:35 Deleting kv.Value: root
2020/11/10 08:26:35 Deleting kv.Value: ubuntu_20.04
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value:
2020/11/10 08:26:35 Deleting kv.Value: vcsosdev.adm.cloud.itzbund.net
2020/11/10 08:26:35 Deleting kv.Value: test
2020/11/10 08:26:35 Creating plugin client for path: /opt/packer_v1.6.5/packer
2020/11/10 08:26:35 Starting plugin: /opt/packer_v1.6.5/packer []string{"/opt/packer_v1.6.5/packer", "plugin", "packer-builder-vsphere-iso"}
2020/11/10 08:26:35 Waiting for RPC address for: /opt/packer_v1.6.5/packer
2020/11/10 08:26:35 [ERR] Checkpoint error: Get "https://checkpoint-api.hashicorp.com/v1/check/packer?arch=amd64&os=linux&signature=8b47c676-adcf-5aa3-f76c-663c696de14e&version=1.6.5": dial tcp: lookup checkpoint-api.hashicorp.com on 10.131.41.12:53: no such host
2020/11/10 08:26:35 packer-builder-vsphere-iso plugin: [INFO] Packer version: 1.6.5 [go1.15.3 linux amd64]
2020/11/10 08:26:35 packer-builder-vsphere-iso plugin: Checking 'PACKER_CONFIG' for a config file path
2020/11/10 08:26:35 packer-builder-vsphere-iso plugin: 'PACKER_CONFIG' not set; checking the default config file path
2020/11/10 08:26:35 packer-builder-vsphere-iso plugin: Attempting to open config file: /root/.packerconfig
2020/11/10 08:26:35 packer-builder-vsphere-iso plugin: [WARN] Config file doesn't exist: /root/.packerconfig
2020/11/10 08:26:35 packer-builder-vsphere-iso plugin: Setting cache directory: /opt/git/packer-config/ubuntu/packer_cache
2020/11/10 08:26:35 Received unix RPC address for /opt/packer_v1.6.5/packer: addr is /tmp/packer-plugin223505664
2020/11/10 08:26:35 packer-builder-vsphere-iso plugin: args: []string{"packer-builder-vsphere-iso"}
2020/11/10 08:26:35 packer-builder-vsphere-iso plugin: Plugin address: unix /tmp/packer-plugin223505664
2020/11/10 08:26:35 packer-builder-vsphere-iso plugin: Waiting for connection...
2020/11/10 08:26:35 packer-builder-vsphere-iso plugin: Serving a plugin connection...
2020/11/10 08:26:35 Creating plugin client for path: /opt/packer_v1.6.5/packer
2020/11/10 08:26:35 Starting plugin: /opt/packer_v1.6.5/packer []string{"/opt/packer_v1.6.5/packer", "plugin", "packer-provisioner-shell"}
2020/11/10 08:26:35 Waiting for RPC address for: /opt/packer_v1.6.5/packer
2020/11/10 08:26:35 packer-builder-vsphere-iso plugin: [ERR] Checkpoint error: Get "https://checkpoint-api.hashicorp.com/v1/check/packer?arch=amd64&os=linux&signature=8b47c676-adcf-5aa3-f76c-663c696de14e&version=1.6.5": dial tcp: lookup checkpoint-api.hashicorp.com on 10.131.41.12:53: no such host
2020/11/10 08:26:35 packer-provisioner-shell plugin: [INFO] Packer version: 1.6.5 [go1.15.3 linux amd64]
2020/11/10 08:26:35 packer-provisioner-shell plugin: Checking 'PACKER_CONFIG' for a config file path
2020/11/10 08:26:35 packer-provisioner-shell plugin: 'PACKER_CONFIG' not set; checking the default config file path
2020/11/10 08:26:35 packer-provisioner-shell plugin: Attempting to open config file: /root/.packerconfig
2020/11/10 08:26:35 packer-provisioner-shell plugin: [WARN] Config file doesn't exist: /root/.packerconfig
2020/11/10 08:26:35 packer-provisioner-shell plugin: Setting cache directory: /opt/git/packer-config/ubuntu/packer_cache
2020/11/10 08:26:35 packer-provisioner-shell plugin: args: []string{"packer-provisioner-shell"}
2020/11/10 08:26:35 Received unix RPC address for /opt/packer_v1.6.5/packer: addr is /tmp/packer-plugin705361351
2020/11/10 08:26:35 packer-provisioner-shell plugin: Plugin address: unix /tmp/packer-plugin705361351
2020/11/10 08:26:35 packer-provisioner-shell plugin: Waiting for connection...
2020/11/10 08:26:35 packer-provisioner-shell plugin: Serving a plugin connection...
2020/11/10 08:26:35 Creating plugin client for path: /opt/packer_v1.6.5/packer
2020/11/10 08:26:35 Starting plugin: /opt/packer_v1.6.5/packer []string{"/opt/packer_v1.6.5/packer", "plugin", "packer-provisioner-file"}
2020/11/10 08:26:35 Waiting for RPC address for: /opt/packer_v1.6.5/packer
2020/11/10 08:26:35 packer-provisioner-shell plugin: [ERR] Checkpoint error: Get "https://checkpoint-api.hashicorp.com/v1/check/packer?arch=amd64&os=linux&signature=8b47c676-adcf-5aa3-f76c-663c696de14e&version=1.6.5": dial tcp: lookup checkpoint-api.hashicorp.com on 10.131.41.12:53: no such host
2020/11/10 08:26:35 packer-provisioner-file plugin: [INFO] Packer version: 1.6.5 [go1.15.3 linux amd64]
2020/11/10 08:26:35 packer-provisioner-file plugin: Checking 'PACKER_CONFIG' for a config file path
2020/11/10 08:26:35 packer-provisioner-file plugin: 'PACKER_CONFIG' not set; checking the default config file path
2020/11/10 08:26:35 packer-provisioner-file plugin: Attempting to open config file: /root/.packerconfig
2020/11/10 08:26:35 packer-provisioner-file plugin: [WARN] Config file doesn't exist: /root/.packerconfig
2020/11/10 08:26:35 packer-provisioner-file plugin: Setting cache directory: /opt/git/packer-config/ubuntu/packer_cache
2020/11/10 08:26:35 packer-provisioner-file plugin: args: []string{"packer-provisioner-file"}
2020/11/10 08:26:35 Received unix RPC address for /opt/packer_v1.6.5/packer: addr is /tmp/packer-plugin253417820
2020/11/10 08:26:35 packer-provisioner-file plugin: Plugin address: unix /tmp/packer-plugin253417820
2020/11/10 08:26:35 packer-provisioner-file plugin: Waiting for connection...
2020/11/10 08:26:35 Creating plugin client for path: /opt/packer_v1.6.5/packer
2020/11/10 08:26:35 Starting plugin: /opt/packer_v1.6.5/packer []string{"/opt/packer_v1.6.5/packer", "plugin", "packer-provisioner-shell"}
2020/11/10 08:26:35 packer-provisioner-file plugin: Serving a plugin connection...
2020/11/10 08:26:35 Waiting for RPC address for: /opt/packer_v1.6.5/packer
2020/11/10 08:26:35 packer-provisioner-file plugin: [ERR] Checkpoint error: Get "https://checkpoint-api.hashicorp.com/v1/check/packer?arch=amd64&os=linux&signature=8b47c676-adcf-5aa3-f76c-663c696de14e&version=1.6.5": dial tcp: lookup checkpoint-api.hashicorp.com on 10.131.41.12:53: no such host
2020/11/10 08:26:35 Received unix RPC address for /opt/packer_v1.6.5/packer: addr is /tmp/packer-plugin129572049
2020/11/10 08:26:35 packer-provisioner-shell plugin: [INFO] Packer version: 1.6.5 [go1.15.3 linux amd64]
2020/11/10 08:26:35 packer-provisioner-shell plugin: Checking 'PACKER_CONFIG' for a config file path
2020/11/10 08:26:35 packer-provisioner-shell plugin: 'PACKER_CONFIG' not set; checking the default config file path
2020/11/10 08:26:35 packer-provisioner-shell plugin: Attempting to open config file: /root/.packerconfig
2020/11/10 08:26:35 packer-provisioner-shell plugin: [WARN] Config file doesn't exist: /root/.packerconfig
2020/11/10 08:26:35 packer-provisioner-shell plugin: Setting cache directory: /opt/git/packer-config/ubuntu/packer_cache
2020/11/10 08:26:35 packer-provisioner-shell plugin: args: []string{"packer-provisioner-shell"}
2020/11/10 08:26:35 packer-provisioner-shell plugin: Plugin address: unix /tmp/packer-plugin129572049
2020/11/10 08:26:35 packer-provisioner-shell plugin: Waiting for connection...
2020/11/10 08:26:35 packer-provisioner-shell plugin: Serving a plugin connection...
2020/11/10 08:26:35 Preparing build: vsphere-iso
2020/11/10 08:26:35 packer-provisioner-shell plugin: [ERR] Checkpoint error: Get "https://checkpoint-api.hashicorp.com/v1/check/packer?arch=amd64&os=linux&signature=8b47c676-adcf-5aa3-f76c-663c696de14e&version=1.6.5": dial tcp: lookup checkpoint-api.hashicorp.com on 10.131.41.12:53: no such host
Debug mode enabled. Builds will not be parallelized.
2020/11/10 08:26:35 Build debug mode: true
2020/11/10 08:26:35 Force build: true
2020/11/10 08:26:35 On error:
vsphere-iso: output will be in this color.
2020/11/10 08:26:35 Debug enabled, so waiting for build to finish: vsphere-iso

2020/11/10 08:26:35 Starting build run: vsphere-iso
2020/11/10 08:26:35 Running builder: vsphere-iso
2020/11/10 08:26:35 [INFO] (telemetry) Starting builder vsphere-iso
==> vsphere-iso: Pausing after run of step 'StepConnect'. Press enter to continue.
2020/11/10 08:26:36 packer-builder-vsphere-iso plugin: No URLs were provided to Step Download. Continuing...
==> vsphere-iso: Pausing after run of step 'StepDownload'. Press enter to continue.
2020/11/10 08:26:37 packer-builder-vsphere-iso plugin: No CD files specified. CD disk will not be made.
==> vsphere-iso: Pausing after run of step 'StepCreateCD'. Press enter to continue.
==> vsphere-iso: Pausing after run of step 'StepRemoteUpload'. Press enter to continue.
==> vsphere-iso: Creating VM...
==> vsphere-iso: Pausing after run of step 'StepCreateVM'. Press enter to continue.
==> vsphere-iso: Customizing hardware...
==> vsphere-iso: Pausing after run of step 'StepConfigureHardware'. Press enter to continue.
==> vsphere-iso: Mounting ISO images...
2020/11/10 08:26:40 packer-builder-vsphere-iso plugin: Check if ISO path is a Content Library path
2020/11/10 08:26:40 packer-builder-vsphere-iso plugin: ISO path not identified as a Content Library path
2020/11/10 08:26:40 packer-builder-vsphere-iso plugin: Using [sos_fas1_01_nfs_vol1_encrypted] ISO/ubuntu-20.04-live-server-amd64.iso as the datastore path
2020/11/10 08:26:40 packer-builder-vsphere-iso plugin: Creating CD-ROM on controller '&{{{} 200 0xc000b16b00 <nil> <nil> <nil> 0 <nil>} 0 []}' with iso '[sos_fas1_01_nfs_vol1_encrypted] ISO/ubuntu-20.04-live-server-amd64.iso'
==> vsphere-iso: Pausing after run of step 'StepAddCDRom'. Press enter to continue.
==> vsphere-iso: Adding configuration parameters...
==> vsphere-iso: Pausing after run of step 'StepConfigParams'. Press enter to continue.
2020/11/10 08:26:41 packer-builder-vsphere-iso plugin: No floppy files specified. Floppy disk will not be made.
==> vsphere-iso: Pausing after run of step 'StepCreateFloppy'. Press enter to continue.
==> vsphere-iso: Pausing after run of step 'StepAddFloppy'. Press enter to continue.
==> vsphere-iso: Pausing after run of step 'StepHTTPIPDiscover'. Press enter to continue.
2020/11/10 08:26:43 packer-builder-vsphere-iso plugin: Found available port: 8080 on IP: 0.0.0.0
==> vsphere-iso: Starting HTTP server on port 8080
==> vsphere-iso: Pausing after run of step 'StepHTTPServer'. Press enter to continue.
==> vsphere-iso: Set boot order temporary...
==> vsphere-iso: Power on VM...
==> vsphere-iso: Pausing after run of step 'StepRun'. Press enter to continue.
==> vsphere-iso: Waiting 3s for boot...
==> vsphere-iso: HTTP server is working at http://10.123.4.200:8080/
==> vsphere-iso: Typing boot command...
2020/11/10 08:26:48 packer-builder-vsphere-iso plugin: Special code '<esc>' found, replacing with: CodeEscape
2020/11/10 08:26:48 packer-builder-vsphere-iso plugin: [INFO] Waiting 1s
2020/11/10 08:26:49 packer-builder-vsphere-iso plugin: Special code '<esc>' found, replacing with: CodeEscape
2020/11/10 08:26:49 packer-builder-vsphere-iso plugin: [INFO] Waiting 1s
2020/11/10 08:26:50 packer-builder-vsphere-iso plugin: Special code '<f6>' found, replacing with: CodeF6
2020/11/10 08:26:50 packer-builder-vsphere-iso plugin: [INFO] Waiting 1s
2020/11/10 08:26:51 packer-builder-vsphere-iso plugin: Special code '<esc>' found, replacing with: CodeEscape
2020/11/10 08:26:52 packer-builder-vsphere-iso plugin: [INFO] Waiting 1s
2020/11/10 08:26:53 packer-builder-vsphere-iso plugin: Special code '<bs>' found, replacing with: CodeDeleteBackspace
2020/11/10 08:26:53 packer-builder-vsphere-iso plugin: Special code '<bs>' found, replacing with: CodeDeleteBackspace
2020/11/10 08:26:53 packer-builder-vsphere-iso plugin: Special code '<bs>' found, replacing with: CodeDeleteBackspace
2020/11/10 08:26:53 packer-builder-vsphere-iso plugin: Special code '<bs>' found, replacing with: CodeDeleteBackspace
2020/11/10 08:26:53 packer-builder-vsphere-iso plugin: Special code '<bs>' found, replacing with: CodeDeleteBackspace
2020/11/10 08:26:53 packer-builder-vsphere-iso plugin: Sending char ' ', code CodeSpacebar, shift false
2020/11/10 08:26:53 packer-builder-vsphere-iso plugin: Sending char 'a', code CodeA, shift false
2020/11/10 08:26:53 packer-builder-vsphere-iso plugin: Sending char 'u', code CodeU, shift false
2020/11/10 08:26:54 packer-builder-vsphere-iso plugin: Sending char 't', code CodeT, shift false
2020/11/10 08:26:54 packer-builder-vsphere-iso plugin: Sending char 'o', code CodeO, shift false
2020/11/10 08:26:54 packer-builder-vsphere-iso plugin: Sending char 'i', code CodeI, shift false
2020/11/10 08:26:54 packer-builder-vsphere-iso plugin: Sending char 'n', code CodeN, shift false
2020/11/10 08:26:54 packer-builder-vsphere-iso plugin: Sending char 's', code CodeS, shift false
2020/11/10 08:26:54 packer-builder-vsphere-iso plugin: Sending char 't', code CodeT, shift false
2020/11/10 08:26:54 packer-builder-vsphere-iso plugin: Sending char 'a', code CodeA, shift false
2020/11/10 08:26:54 packer-builder-vsphere-iso plugin: Sending char 'l', code CodeL, shift false
2020/11/10 08:26:54 packer-builder-vsphere-iso plugin: Sending char 'l', code CodeL, shift false
2020/11/10 08:26:55 packer-builder-vsphere-iso plugin: Sending char ' ', code CodeSpacebar, shift false
2020/11/10 08:26:55 packer-builder-vsphere-iso plugin: Sending char 'd', code CodeD, shift false
2020/11/10 08:26:55 packer-builder-vsphere-iso plugin: Sending char 's', code CodeS, shift false
2020/11/10 08:26:55 packer-builder-vsphere-iso plugin: Sending char '=', code CodeEqualSign, shift false
2020/11/10 08:26:55 packer-builder-vsphere-iso plugin: Sending char 'n', code CodeN, shift false
2020/11/10 08:26:55 packer-builder-vsphere-iso plugin: Sending char 'o', code CodeO, shift false
2020/11/10 08:26:55 packer-builder-vsphere-iso plugin: Sending char 'c', code CodeC, shift false
2020/11/10 08:26:55 packer-builder-vsphere-iso plugin: Sending char 'l', code CodeL, shift false
2020/11/10 08:26:56 packer-builder-vsphere-iso plugin: Sending char 'o', code CodeO, shift false
2020/11/10 08:26:56 packer-builder-vsphere-iso plugin: Sending char 'u', code CodeU, shift false
2020/11/10 08:26:56 packer-builder-vsphere-iso plugin: Sending char 'd', code CodeD, shift false
2020/11/10 08:26:56 packer-builder-vsphere-iso plugin: Sending char '-', code CodeHyphenMinus, shift false
2020/11/10 08:26:56 packer-builder-vsphere-iso plugin: Sending char 'n', code CodeN, shift false
2020/11/10 08:26:56 packer-builder-vsphere-iso plugin: Sending char 'e', code CodeE, shift false
2020/11/10 08:26:56 packer-builder-vsphere-iso plugin: Sending char 't', code CodeT, shift false
2020/11/10 08:26:56 packer-builder-vsphere-iso plugin: Sending char ';', code CodeSemicolon, shift false
2020/11/10 08:26:56 packer-builder-vsphere-iso plugin: Sending char 's', code CodeS, shift false
2020/11/10 08:26:57 packer-builder-vsphere-iso plugin: Sending char 'e', code CodeE, shift false
2020/11/10 08:26:57 packer-builder-vsphere-iso plugin: Sending char 'e', code CodeE, shift false
2020/11/10 08:26:57 packer-builder-vsphere-iso plugin: Sending char 'd', code CodeD, shift false
2020/11/10 08:26:57 packer-builder-vsphere-iso plugin: Sending char 'f', code CodeF, shift false
2020/11/10 08:26:57 packer-builder-vsphere-iso plugin: Sending char 'r', code CodeR, shift false
2020/11/10 08:26:57 packer-builder-vsphere-iso plugin: Sending char 'o', code CodeO, shift false
2020/11/10 08:26:57 packer-builder-vsphere-iso plugin: Sending char 'm', code CodeM, shift false
2020/11/10 08:26:57 packer-builder-vsphere-iso plugin: Sending char '=', code CodeEqualSign, shift false
2020/11/10 08:26:58 packer-builder-vsphere-iso plugin: Sending char 'h', code CodeH, shift false
2020/11/10 08:26:58 packer-builder-vsphere-iso plugin: Sending char 't', code CodeT, shift false
2020/11/10 08:26:58 packer-builder-vsphere-iso plugin: Sending char 't', code CodeT, shift false
2020/11/10 08:26:58 packer-builder-vsphere-iso plugin: Sending char 'p', code CodeP, shift false
2020/11/10 08:26:58 packer-builder-vsphere-iso plugin: Sending char ':', code CodeSemicolon, shift true
2020/11/10 08:26:58 packer-builder-vsphere-iso plugin: Sending char '/', code CodeSlash, shift false
2020/11/10 08:26:58 packer-builder-vsphere-iso plugin: Sending char '/', code CodeSlash, shift false
2020/11/10 08:26:58 packer-builder-vsphere-iso plugin: Sending char '1', code Code1, shift false
2020/11/10 08:26:58 packer-builder-vsphere-iso plugin: Sending char '0', code Code0, shift false
2020/11/10 08:26:59 packer-builder-vsphere-iso plugin: Sending char '.', code CodeFullStop, shift false
2020/11/10 08:26:59 packer-builder-vsphere-iso plugin: Sending char '1', code Code1, shift false
2020/11/10 08:26:59 packer-builder-vsphere-iso plugin: Sending char '2', code Code2, shift false
2020/11/10 08:26:59 packer-builder-vsphere-iso plugin: Sending char '3', code Code3, shift false
2020/11/10 08:26:59 packer-builder-vsphere-iso plugin: Sending char '.', code CodeFullStop, shift false
2020/11/10 08:26:59 packer-builder-vsphere-iso plugin: Sending char '4', code Code4, shift false
2020/11/10 08:26:59 packer-builder-vsphere-iso plugin: Sending char '.', code CodeFullStop, shift false
2020/11/10 08:26:59 packer-builder-vsphere-iso plugin: Sending char '2', code Code2, shift false
2020/11/10 08:27:00 packer-builder-vsphere-iso plugin: Sending char '0', code Code0, shift false
2020/11/10 08:27:00 packer-builder-vsphere-iso plugin: Sending char '0', code Code0, shift false
2020/11/10 08:27:00 packer-builder-vsphere-iso plugin: Sending char ':', code CodeSemicolon, shift true
2020/11/10 08:27:00 packer-builder-vsphere-iso plugin: Sending char '8', code Code8, shift false
2020/11/10 08:27:00 packer-builder-vsphere-iso plugin: Sending char '0', code Code0, shift false
2020/11/10 08:27:00 packer-builder-vsphere-iso plugin: Sending char '8', code Code8, shift false
2020/11/10 08:27:00 packer-builder-vsphere-iso plugin: Sending char '0', code Code0, shift false
2020/11/10 08:27:00 packer-builder-vsphere-iso plugin: Sending char '/', code CodeSlash, shift false
2020/11/10 08:27:00 packer-builder-vsphere-iso plugin: Special code '<enter>' found, replacing with: CodeReturnEnter
2020/11/10 08:27:01 packer-builder-vsphere-iso plugin: [INFO] Waiting 1s
==> vsphere-iso: Pausing after run of step 'boot_command: <esc><wait><esc><wait><f6><wait><esc><wait><bs><bs><bs><bs><bs> autoinstall ds=nocloud-net;seedfrom=http://10.123.4.200:8080/<enter><wait>'. Press enter to continue.
==> vsphere-iso: Pausing after run of step 'StepBootCommand'. Press enter to continue.
2020/11/10 08:27:06 packer-builder-vsphere-iso plugin: [INFO] Waiting for IP, up to total timeout: 30m0s, settle timeout: 5m0s
==> vsphere-iso: Waiting for IP...
==> vsphere-iso: Pausing before cleanup of step 'StepBootCommand'. Press enter to continue.
==> vsphere-iso: Pausing before cleanup of step 'StepRun'. Press enter to continue.
==> vsphere-iso: Clear boot order...
==> vsphere-iso: Power off VM...
==> vsphere-iso: Pausing before cleanup of step 'StepHTTPServer'. Press enter to continue.
==> vsphere-iso: Pausing before cleanup of step 'StepHTTPIPDiscover'. Press enter to continue.
==> vsphere-iso: Pausing before cleanup of step 'StepAddFloppy'. Press enter to continue.
==> vsphere-iso: Pausing before cleanup of step 'StepCreateFloppy'. Press enter to continue.
==> vsphere-iso: Pausing before cleanup of step 'StepConfigParams'. Press enter to continue.
==> vsphere-iso: Pausing before cleanup of step 'StepAddCDRom'. Press enter to continue.
==> vsphere-iso: Pausing before cleanup of step 'StepConfigureHardware'. Press enter to continue.
==> vsphere-iso: Pausing before cleanup of step 'StepCreateVM'. Press enter to continue.
==> vsphere-iso: Destroying VM...
==> vsphere-iso: Pausing before cleanup of step 'StepRemoteUpload'. Press enter to continue.
==> vsphere-iso: Pausing before cleanup of step 'StepCreateCD'. Press enter to continue.
==> vsphere-iso: Pausing before cleanup of step 'StepDownload'. Press enter to continue.
==> vsphere-iso: Pausing before cleanup of step 'StepConnect'. Press enter to continue.
2020/11/10 08:28:06 [INFO] (telemetry) ending vsphere-iso
2020/11/10 08:28:06 Waiting on builds to complete...
==> Wait completed after 1 minute 30 seconds
2020/11/10 08:28:06 machine readable: error-count []string{"1"}
==> Some builds didn't complete successfully and had errors:
2020/11/10 08:28:06 machine readable: vsphere-iso,error []string{"unable to find an IP"}
==> Builds finished but no artifacts were created.
Build 'vsphere-iso' errored after 1 minute 30 seconds: unable to find an IP

==> Wait completed after 1 minute 30 seconds

==> Some builds didn't complete successfully and had errors:
--> vsphere-iso: unable to find an IP

==> Builds finished but no artifacts were created.
2020/11/10 08:28:06 Cancelling builder after context cancellation context canceled
2020/11/10 08:28:06 [INFO] (telemetry) Finalizing.
2020/11/10 08:28:06 [WARN] (telemetry) Error finalizing report. This is safe to ignore. Post "https://checkpoint-api.hashicorp.com/v1/telemetry/packer": dial tcp: lookup checkpoint-api.hashicorp.com on 10.131.41.12:53: no such host
2020/11/10 08:28:06 waiting for all plugin processes to complete...
2020/11/10 08:28:06 /opt/packer_v1.6.5/packer: plugin process exited
2020/11/10 08:28:06 /opt/packer_v1.6.5/packer: plugin process exited
2020/11/10 08:28:06 /opt/packer_v1.6.5/packer: plugin process exited
2020/11/10 08:28:06 /opt/packer_v1.6.5/packer: plugin process exited

````

**Comment by SwampDragons** [comment]
Can you please share a minimal complete config that can be used to reproduce the issue?

**Comment by Runkar** [comment]
My bad...

Host config: RHEL 8.2, 2 CPUs, 4GB RAM
Packer config:
````
{
  "_comment": "{{user `comment`}}",
  "builders": [
    {
      "CPUs": "{{user `core_count`}}",
      "RAM": "{{user `ram_size`}}",
      "boot_command": [
        "<esc><wait><esc><wait><f6><wait><esc><wait>",
        "<bs><bs><bs><bs><bs>",
        " autoinstall ds=nocloud-net;seedfrom=http://{{ .HTTPIP }}:{{ .HTTPPort }}/",
        "<enter><wait>"
      ],
      "boot_order": "{{user `boot_order`}}",
      "boot_wait": "{{user `boot_wait`}}",
      "cluster": "{{user `cluster`}}",
      "configuration_parameters": [
        {
          "disk.EnableUUID": "true"
        }
      ],
      "convert_to_template": "true",
      "cpu_hot_plug": "{{user `cpu_hotplug`}}",
      "datastore": "{{user `datastore`}}",
      "disk_controller_type": "{{user `disk_controller`}}",
      "firmware": "{{user `firmware`}}",
      "folder": "Templates/Packer/{{user `folder`}}",
      "guest_os_type": "{{user `os_type`}}",
      "http_directory": "{{user `http_directory`}}",
      "http_port_min": "8080",
      "http_port_max": "8080",
      "insecure_connection": "{{user `insecure`}}",
      "iso_paths": [
        "{{user `os_iso`}}"
      ],
      "network_adapters": [
        {
          "network": "{{user `network`}}",
          "network_card": "{{user `network_card`}}"
        }
      ],
      "notes": "{{user `notes`}}",
      "password": "{{user `vcenter_pass`}}",
      "ram_hot_plug": "{{user `ram_hotplug`}}",
      "ip_settle_timeout": "{{user `ip_settle`}}",
      "ssh_password": "{{user `ssh_pass`}}",
      "ssh_timeout": "{{user `ssh_timeout`}}",
      "ssh_username": "{{user `ssh_user`}}",
      "storage": [
        {
          "disk_size": "{{user `disk_size1`}}"
        },
        {
          "disk_size": "{{user `disk_size2`}}",
          "disk_thin_provisioned": "{{user `disk_provision`}}"
        }
      ],
      "type": "vsphere-iso",
      "username": "{{user `vcenter_login`}}",
      "vcenter_server": "{{user `vcenter_server`}}",
      "vm_name": "{{user `vm_name`}}",
      "vm_version": "11"
    }
  ],
  "description": "{{user `description`}}",
  "min_packer_version": "1.6.0",
  "provisioners": [
    {
      "execute_command": "/bin/bash {{.Path}} {{user `ubuntu_minor`}}",
      "expect_disconnect": true,
      "scripts": [
        "{{user `apt_update`}}",
        "{{user `post_install_config_file`}}"
      ],
      "type": "shell"
    },
    {
      "destination": "/packer/Packages/{{user `folder`}}/{{user `vm_name`}}_packages.txt",
      "direction": "download",
      "source": "/tmp/packages.txt",
      "type": "file"
    },
    {
      "execute_command": "/bin/bash {{.Path}} {{user `ubuntu_minor`}}",
      "scripts": [
        "{{user `post_install_cleanup_file`}}"
      ],
      "type": "shell"
    }
  ],
  "sensitive-variables": [
    "vcenter_login",
    "vcenter_pass",
    "ssh_user",
    "ssh_pass"
  ]
}
````

Variables file:

````
{
  "comment": "Autor: Jens Schütz, erstellt am: 04.05.2020, zuletzte geändert am: 04.05.2020",
  "description": "Variablen für Packer-Datei zur automatisierten Erstellung des Ubuntu 20.04 Templates",
  "ubuntu_minor": "ubuntu_20.04",
  "version": "",
  "vcenter_server": "vcsosdev.adm.cloud.itzbund.net",
  "insecure": "true",
  "vcenter_login": "packer@adm.cloud.itzbund.local",
  "vcenter_pass": "",
  "folder": "test",
  "cluster": "clashare1003",
  "datastore": "sos_fas1_01_nfs_vol1_encrypted",
  "vm_name": "Template_Ubuntu_20.04_v{{user `version`}}",
  "notes": "Ubuntu Linux 20.04\n\n- erstellt: {{isotime \"02.01.2006\"}}\n- 512MB \/ 10GB HDD\n- SSH aktiv\n- open-vm-tools, jq, chrony",
  "core_count": "1",
  "cpu_hotplug": "true",
  "ram_size": "2048",
  "ram_hotplug": "true",
  "disk_size1": "512",
  "disk_size2": "10240",
  "os_type": "ubuntu64Guest",
  "firmware": "bios",
  "disk_controller": "pvscsi",
  "disk_provision": "true",
  "network": "DEV99|PACKER|PACKER_EPG",
  "network_card": "vmxnet3",
  "boot_wait": "3s",
  "http_directory": "/opt/git/packer-config/ubuntu/ubuntu_20.04/http",
  "ssh_user": "root",
  "ssh_pass": "",
  "ssh_timeout": "15m",
  "ip_settle": "10m",
  "os_iso": "[sos_fas1_01_nfs_vol1_encrypted] ISO/ubuntu-20.04-live-server-amd64.iso",
  "apt_update": "/opt/git/packer-config/ubuntu/bash_scripts/apt_update.sh",
  "post_install_config_file": "/opt/git/packer-config/ubuntu/bash_scripts/post_install_config.sh",
  "post_install_cleanup_file": "/opt/git/packer-config/ubuntu/bash_scripts/post_install_cleanup.sh"
}
````

Cloud-init config:
````
#cloud-config
autoinstall:
  version: 1
  locale: en_US
  keyboard:
    layout: de
    variant: de
  network:
    network:
      version: 2
      ethernets:
        ens160:
          dhcp4: true
  storage:
    config:
    - {ptable: gpt, path: /dev/sda, preserve: false, name: '', grub_device: true,
      type: disk, id: disk-sda}
    - {path: /dev/sdb, wipe: superblock, preserve: false, name: '', grub_device: false,
      type: disk, id: disk-sdb}
    - {device: disk-sda, size: 1048576, flag: bios_grub, number: 1, preserve: false,
      type: partition, id: partition-0}
    - {device: disk-sda, size: 524288000, wipe: superblock, flag: '', number: 2, preserve: false,
      type: partition, id: partition-1}
    - {fstype: ext4, volume: partition-1, preserve: false, type: format, id: format-0}
    - name: vg0
      devices: [disk-sdb]
      preserve: false
      type: lvm_volgroup
      id: lvm_volgroup-0
    - {name: lv-root, volgroup: lvm_volgroup-0, size: 6979321856B, preserve: false,
      type: lvm_partition, id: lvm_partition-0}
    - {fstype: xfs, volume: lvm_partition-0, preserve: false, type: format, id: format-1}
    - {device: format-1, path: /, type: mount, id: mount-1}
    - {name: lv-var, volgroup: lvm_volgroup-0, size: 2147483648B, preserve: false,
      type: lvm_partition, id: lvm_partition-1}
    - {fstype: xfs, volume: lvm_partition-1, preserve: false, type: format, id: format-2}
    - {device: format-2, path: /var, type: mount, id: mount-2}
    - {name: lv-home, volgroup: lvm_volgroup-0, size: 1073741824B, preserve: false,
      type: lvm_partition, id: lvm_partition-2}
    - {fstype: xfs, volume: lvm_partition-2, preserve: false, type: format, id: format-3}
    - {device: format-3, path: /home, type: mount, id: mount-3}
    - {device: format-0, path: /boot, type: mount, id: mount-0}
  ssh:
    install-server: yes
  user-data:
    apt:
      http_proxy: 'http://10.130.165.20:3128/'
      https_proxy: 'http://10.130.165.20:3128/'
    packages:
      - open-vm-tools
    timezone: Europe/Berlin
    disable_root: false
    chpasswd:
      expire: false
      list:
        - root:$6$erG4iB2NK0$EDl2hp6w820SI069m9cu1tY8jqA5wkyn//keQR6xvoI/L7pC7m9BIzaFgLA4T0uvemMJx5UeH22ujzyJbJwM81
  ssh_pwauth: true
  late-commands:
    - 'sed -i "s/dhcp4: true/&\n      dhcp-identifier: mac/" /target/etc/netplan/00-installer-config.yaml'
    - 'sed -i "s/^PermitRootLogin.*//g" /target/etc/ssh/sshd_config'
    - 'sed -i "s/^#PermitRootLogin.*/PermitRootLogin yes/" /target/etc/ssh/sshd_config'
````

**Comment by Runkar** [comment]
Update 2020/11/30:

With SLES it is the same: build stops while autoyast installation is in progress. Reproducable in packer v1.6.5.
With packer v1.6.0 the run succeeds. So the fault is not OS specific.

**Comment by SwampDragons** [comment]
Hi, thanks for reaching out. If this is on the VSphere builder, I think the issue may be a duplicate of #9973 and is potentially solved by the PR in #10273 once we can resolve conflicts there. I'll try to push a fix to get that branch to build so you can test the fix.

**Comment by SwampDragons** [comment]
Artifacts from that Pr can be found in CircleCI at https://app.circleci.com/pipelines/github/hashicorp/packer/8101/workflows/d79e5a3e-c0d1-448e-93b6-8025d4cc7eeb/jobs/94230/artifacts

**Comment by Runkar** [comment]
Hey, thank you for your support!

You are right, this is on vsphere-iso builder.

I used your v1.6.6-dev version for testing but unfortunately the build now breaks even earlier (on creating the VM) with the error:
```vsphere-iso,error []string{"error creating vm: Invalid configuration for device '2'."}```

According to the KB article (https://kb.vmware.com/s/article/65205) this could be vmware related issue. However the ESXi/vCenter used is on v6.7 U3 and additionally setting "vm_version" to 14 and giving a max vram ""video_ram": "64000" didn't solve the error.

**Comment by jhawk28** [comment]
@SwampDragons actually, I dont think the issue has anything to do with the http_ip. The issue is this: ```return "", fmt.Errorf("unable to find an IP")```.

The code is expecting that if no IP was found, an empty trying is to be returned. We are returning an empty string and an error. If we return nil, the wait for IP logic will keep looping until an IP is returned.

**Comment by jhawk28** [comment]
@Runkar if you want to test, artifacts are here: https://app.circleci.com/pipelines/github/hashicorp/packer/8113/workflows/72e4fff5-05fd-420d-8067-9c5c6a8e4cfa/jobs/94389/artifacts
